btraced.php

```diff
@@ -147,7 +147,7 @@ function parseMilli($str)
  
  $strJSON = "{\"id\":0,\"tripid\":".$tripid.",".$strJSONPoints."\"valid\":true";
  if ($getTripURL == "1")
-     $strJSON .= ",\"tripurl\":\"http%3a%2f%2fglidingops%2ecom%2f\"";
+     $strJSON .= ",\"tripurl\":\"https%3a%2f%2fglidingops%2ecom%2f\"";
```

Comment:
You could have used `urlencode` here, to save you some trouble of manually taking care of special characters.
http://php.net/manual/en/function.urlencode.php

Reply:
Yep, I wrote this some time back beodre I new about urlencode.  However, I must say I have some problems with the php implementation of it and have been wary of using it.  In this case I agree it would be fine.  But its ok as is so best not change too much.  
I will merge and put in tonight,